### PR TITLE
Fix lab UU documentation

### PR DESCRIPTION
**Pull request by chessvivek**
Add a few lines to tell the students about existence and significance of the given env-reference executable

### CODE REVIEW

File: _docs/utilities_unleashed.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ $
 
 This has the exact same behavior as before, because `TEMP` is first set to `EST5EDT`, and then when `TZ` is set to `%TEMP`, the value of `EST5EDT` is retrieved and then `TZ` is set to that. Notice that the variables are set sequentially, or else it wouldn't work.
 
+We have provided you with a reference executable `env-reference` for you to test your understanding of `env`'s expected behavior. Kindly verify the behavior of your `env` against that of `env-reference` on a variety of test cases to avoid unnecesarily losing points.
```
Comment (Chaldron) [inline _docs/utilities_unleashed.md:132]: I would replace the second sentence with something like "You can use `env-reference` to see if you `env`'s behavior matches the expected output" or something like that. Other than that, this looks solid.

**Review by Chaldron — CHANGES_REQUESTED** [review]

File: _docs/utilities_unleashed.md (hunk 1 above)
Comment (Chaldron) [reply _docs/utilities_unleashed.md:132]: Also, make sure to add a description to the pull request to identify what exactly you're changing.

Comment (chessvivek) [reply _docs/utilities_unleashed.md]: Added the description, and updated that sentence.

**Review by Chaldron — APPROVED** [review]
LGTM

**Comment by MaciejKrzysiak** [comment]
[Update UU Utterances](https://trello.com/c/zssctSdg/66-update-uu-utterances)

**Pull request merged**